### PR TITLE
Remove old _FORCE_NO_WCSAXES variable

### PR DESCRIPTION
**Pull request by dstansby**
I noticed that `_FORCE_NO_WCSAXES` doesn't seem to be used anywhere in the tests, so remove it as I presume it's old and uneeded? ping @Cadair who put this in 6 yrs ago in case I'm missing something.

### CODE REVIEW

**Comment by Cadair** [comment]
@dstansby can you update this?

**Review by nabobalis — APPROVED** [review]

File: sunpy/visualization/wcsaxes_compat.py [hunk 1]
```diff
@@ -29,10 +26,7 @@ def is_wcsaxes(axes):
     `bool`
         Result of the test.
     """
-    if not _FORCE_NO_WCSAXES:
-        return isinstance(axes, wcsaxes.WCSAxes)
-    else:
-        return False
+    return isinstance(axes, wcsaxes.WCSAxes)
```
Comment (nabobalis) [inline sunpy/visualization/wcsaxes_compat.py:29]: Not sure we need this method anymore but we can punt down the road.

File: sunpy/visualization/wcsaxes_compat.py [hunk 2]
```diff
@@ -6,9 +6,6 @@
 import astropy.units as u
 from astropy.visualization import wcsaxes
 
-# Force is put here to enable disabling all checks in this module.
-# It should only be used by tests and other such hacks.
-_FORCE_NO_WCSAXES = False
```
Comment (nabobalis) [inline sunpy/visualization/wcsaxes_compat.py:9]:
```suggestion
```

File: sunpy/visualization/wcsaxes_compat.py (hunk 1 above)
Comment (dstansby) [reply sunpy/visualization/wcsaxes_compat.py:28]: Yeah, might as well wait till 3.0 since a deprecation needs to be in one LTS anyway.

**Pull request merged**